acceptance_modelisation/base_acceptance_map_creator.py

```diff
@@ -597,15 +663,22 @@ def create_acceptance_map_cos_zenith_binned(self,
         return acceptance_map
 
     def create_acceptance_map_cos_zenith_interpolated(self,
-                                                      observations: Observations
+                                                      observations: Observations,
+                                                      off_observations: Observations = None,
+                                                      base_model: dict[Any, BackgroundIRF] = None
```

Comment:
Should it be `dict[float, BackgroundIRF]`?

Reply:
New BackgroundCollectionZenith object solve this point.